db/schema.rb

```diff
@@ -27,6 +27,7 @@
     t.string "addressable_type"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.string "county", null: false
```

Comment:
Looking at the other address fields and knowing we can skip the validation from SmartyStreets I don’t think we should set a null constraint here.